activestorage/active.py

```diff
@@ -310,10 +307,6 @@ def _get_selection(self, *args):
         # hopefully fix pyfive to get a dtype directly
```

Comment:
I am not sure the docstring for this method, and this comment are quite right any more.

Reply:
done in 4cdeb1b17ec8bbe116d261bc0f603a040e2cb22c